ground/src/main/java/com/google/android/ground/model/geometry/MultiPolygon.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.ground.model.geometry
+
+import org.locationtech.jts.geom.GeometryFactory
+import org.locationtech.jts.geom.MultiPolygon
+
+/** A grouping of one or more [Polygon]s. */
+data class MultiPolygon(val polygons: Collection<Polygon>) : Geometry {
+    private val factory: GeometryFactory = GeometryFactory()
+    private val geometries = polygons.map { it.geometry }
+
+    override val geometry: MultiPolygon = factory.createMultiPolygon(geometries.toTypedArray())
+    override val type: GeometryType = GeometryType.MULTIPOLYGON
+    override val isClosed: Boolean = true
```

Comment:
What is this used for? Did it leak from the polygon drawing implementation? If so, whether the shape needs to be a LinearRing/Polygon or LineString is a detail that should be handle in the ViewModel, not in the actual model, IMO.

Reply:
Ok, stgm -- I was approaching it from the opposite angle (since closure is a property of the geometry itself not a property of the viewmodel, I felt it belonged in the geometry, but it works either way, esp. if we just use "ephemeral" incomplete objects in the VM)

Reply:
I see.. I think the key is that closed rings are modeled as different classes from open rings altogether, since they're defined as different geometry types by OGC. Contradicting the standard definition of a polygon by allowing the to be LineStrings as well may (will?) lead to confusion, so it's probably better to be consistent. Thanks for all the updates!